### ISSUE TITLE
[Help!!] how to use own object detection model..?

### ISSUE THREAD

**Issue by YouMinJung** [Syn-McJ/TFClassify-Unity#32]
I'm not familiar to tensorflow and unity... 
I wanna use pre-trained tensorflow model in zoo. (ssd mobilenet v2 coco)
But there are 5 files in coco model. (checkpoint, frozen_inference_gragh.pb, model.ckpt.data-00000-of-00001, model.ckpt.index, model.ckpt.meta)

I wanna .pb to bytes... but I don't know how to convert it. 
I searched the information using google. It was told to use the freeze_gragh. I tried to use it. but I don't have .ckpt file... (I only have .ckpt.meta etc..) 
How can I convert the file...?

ps. Exectly, What kind of model did you uesd?

**Comment by Syn-McJ** [comment]
Hi @YouMinJung,
I used ssd_mobilenet from [TensorFlow examples](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android).

You only need 2 files - frozen_inference_gragh.pb and file with labels. You simply have to rename the file to change .pb extension to .bytes, no need to convert anything.

Be aware that, as I mentioned in README, the performance of Detect sample isn't very good.
